css/src/components/progress-bar.scss

```diff
@@ -1,6 +1,6 @@
 $progress-bar-background-color: $secondary-background-glow-high-contrast !default;
 $progress-bar-value-background-color: $primary !default;
-$progress-bar-value-background-color-warning: $warning !default;
+$progress-bar-value-background-color-warning: $colorPaletteDarkOrangeBackground3 !default;
```

Comment:
`$colorPaletteDarkOrangeBackground3` actually refers to the theme map, it just doesn't have a semantic name. Fluent has a semantic warning color, but they also use non-semantically named colors. For this instance, they chose to use a non-semantic color instead of their standard warning colors. The challenge here is that there is a lack of consistency at times. Perhaps I can set this color as `warning-alternate`.